src/templates/dataGrid.jsx

```diff
@@ -271,72 +85,6 @@ class DataGrid extends React.Component {
             [`${classNamePrefix}-sticky_columns`]: isStickyColumns,
         });
 
-        if (isStickyColumns) {
-            return (
-                <div
-                    className={containerClasses}
-                    style={style}
-                >
-                    <div
-                        className={`${classNamePrefix}_fixed_body`}
-                        onScroll={this.onScroll}
-                    >
-                        <DataGridTable
-                            bleed={bleed}
-                            className={className}
-                            classNamePrefix={classNamePrefix}
-                            collapsed={collapsed}
-                            columns={columns}
-                            data={data}
-                            fontSize={fontSize}
-                            handle={handle}
-                            id={id}
-                            idPrefix="body"
-                            minWidth={minWidth}
-                            moduleType={moduleType}
-                            ref={(ref) => { this.bodyTable = ref; }}
-                            rowProps={rowProps}
-                            size={size}
-                            sizes={sizes}
-                            stickyColumnWidth={stickyColumnWidth}
-                            stickyColumns={stickyColumns}
-                            stretch={stretch}
-                            style={{
-                                minWidth,
-                            }}
-                        />
-                    </div>
-
-                    <div className={`${classNamePrefix}_fixed_column`}>
-                        <DataGridTable
-                            bleed={bleed}
-                            className={className}
-                            classNamePrefix={classNamePrefix}
-                            columns={_.slice(columns, 0, stickyColumns)}
-                            data={data}
-                            dropShadow={scrolledRight}
-                            fontSize={fontSize}
-                            handle={handle}
-                            id={id}
-                            idPrefix="column"
-                            minWidth={minWidth}
-                            moduleType={moduleType}
-                            onSplitter={this.onSplitterClick}
-                            onSplitterDragEnd={this.onSplitterDragEnd}
-                            rowProps={rowProps}
-                            size={size}
-                            sizes={sizes}
-                            stickyColumnWidth={stickyColumnWidth}
-                            stickyColumns={stickyColumns}
-                            stretch={stretch}
-                        />
-                    </div>
-
-                    <div className={`${classNamePrefix}_fixed_bottom`} />
-                </div>
-            );
-        }
-
         return (
```

Comment:
So, are you saying that you want to go through the few places in `church-management` repo and replace sticky column logic there by stripping the old methods out of this component?

Reply:
@War777 answered/explained what is going to happen during a linger and I love being able to remove these methods!

Reply:
The table rendered by `<DataGridTable />` is now in charge of handling sticky behavior, so these methods are not neccesary anymore, there are just 5 places where `<DataGrid />` in the HC Client, I tested and it seems to be working correctly.